RecoCTPPS/TotemRPLocal/interface/CTPPSTimingTrackRecognition.h

```diff
@@ -191,34 +200,49 @@ CTPPSTimingTrackRecognition<TRACK_TYPE, HIT_TYPE>::getHitSpatialRange(const HitV
   SpatialRange result;
 
   for (const auto& hit : hits) {
-    const float xBegin = hit.getX() - 0.5f*hit.getXWidth();
-    const float yBegin = hit.getY() - 0.5f*hit.getYWidth();
-    const float zBegin = hit.getZ() - 0.5f*hit.getZWidth();
-    const float xEnd = hit.getX() + 0.5f*hit.getXWidth();
-    const float yEnd = hit.getY() + 0.5f*hit.getYWidth();
-    const float zEnd = hit.getZ() + 0.5f*hit.getZWidth();
+    const float xBegin = hit.getX() - 0.5f*hit.getXWidth(), xEnd = hit.getX() + 0.5f*hit.getXWidth();
+    const float yBegin = hit.getY() - 0.5f*hit.getYWidth(), yEnd = hit.getY() + 0.5f*hit.getYWidth();
+    const float zBegin = hit.getZ() - 0.5f*hit.getZWidth(), zEnd = hit.getZ() + 0.5f*hit.getZWidth();
 
     if (!initialized) {
       result.xBegin = xBegin;
-      result.yBegin = yBegin;
-      result.zBegin = zBegin;
       result.xEnd = xEnd;
+      result.yBegin = yBegin;
       result.yEnd = yEnd;
+      result.zBegin = zBegin;
       result.zEnd = zEnd;
       initialized = true;
       continue;
     }
-    if (xBegin < result.xBegin) result.xBegin = xBegin;
-    if (yBegin < result.yBegin) result.yBegin = yBegin;
-    if (zBegin < result.zBegin) result.zBegin = zBegin;
-
-    if (xEnd > result.xEnd) result.xEnd = xEnd;
-    if (yEnd > result.yEnd) result.yEnd = yEnd;
-    if (zEnd > result.zEnd) result.zEnd = zEnd;
+    result.xBegin = std::min(result.xBegin, xBegin);
+    result.xEnd = std::max(result.xEnd, xEnd);
+    result.yBegin = std::min(result.yBegin, yBegin);
+    result.yEnd = std::max(result.yEnd, yEnd);
+    result.zBegin = std::min(result.zBegin, zBegin);
+    result.zEnd = std::max(result.zEnd, zEnd);
   }
 
   return result;
 }
 
+template<class TRACK_TYPE, class HIT_TYPE> inline
+bool
+CTPPSTimingTrackRecognition<TRACK_TYPE, HIT_TYPE>::timeEval(const HitVector& hits, float& mean_time, float& time_sigma) const
+{
+  float mean_num = 0.f, mean_denom = 0.f;
+  bool valid_hits = false;
+  for (const auto& hit : hits) {
+    if (hit.getTPrecision() <= 0.)
+      continue;
+    const float weight = std::pow(hit.getTPrecision(), -2);
```

Comment:
header consistency check flags the use of ```pow``` as problematic.
Please include ```<cmath>```

Reply:
Resolved in https://github.com/cms-sw/cmssw/pull/26327/commits/9e32e5ffde85854a01410ebb0c180073996fdb5d